test/libp2p.spec.js

```diff
@@ -415,6 +415,15 @@ describe('libp2p-ipfs', () => {
     }
   })
 
+  it('nodeE ping to nodeF', (done) => {
+    const p = nodeE.ping(nodeF.peerInfo)
+
+    p.on('ping', (time) => {
+      p.stop()
+      done()
+    })
+  })
```

Comment:
It does only send one ping? Something tells me this event will be fired up a bunch of times and the only thing preventing the test from failing is that it is the last test, so it closes soon enough